src/controllers/expensesController.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const { StatusCodes, ReasonPhrases } = require('http-status-codes');
+const expensesService = require('../services/expensesService');
+const usersService = require('../services/usersService');
+
+function get(req, res) {
+  const { userId, categories, from, to } = req.query;
+
+  res.status(StatusCodes.OK).send(
+    expensesService.get({
+      userId,
+      categories,
+      from,
+      to,
+    }),
+  );
+}
+
+function post(req, res) {
+  const data = req.body;
```

Comment:
The `post` function does not validate the structure of `data` before using it. Ensure that `data` contains all necessary fields and is correctly formatted before proceeding with the operation.